app/code/core/Mage/Eav/Model/Config/Source/Form.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Maho
+ *
+ * @category   Mage
+ * @package    Mage_Eav
+ * @copyright  Copyright (c) 2006-2020 Magento, Inc. (https://magento.com)
+ * @copyright  Copyright (c) 2024 Maho (https://mahocommerce.com)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Used in creating options for use_in_forms selection
+ *
+ * @category   Mage
+ * @package    Mage_Eav
+ */
+class Mage_Eav_Model_Config_Source_Form
+{
+    protected const XML_PATH_EAV_FORMS = 'global/eav_forms';
+
+    public function toOptionArray(string $entity = ''): array
+    {
+        $node = Mage::getConfig()->getNode(self::XML_PATH_EAV_FORMS . '/' . $entity);
+        if ($node === false) {
```

Comment:
can $node be false? it seems ->getNode() never returns false

Reply:
Yes, if an empty string is passed for `$entity`:
```php
var_dump(Mage::getConfig()->getNode('global/eav_forms/')); 
// Outputs: bool(false)
```

Reply:
right, then the docblock for the getNode() is wrong, let's see